ros_control.rosinstall

```diff
@@ -1,5 +1,7 @@
 - git: {local-name: ros_control, uri: 'https://github.com/ros-controls/ros_control.git', version: jade-devel}
 - git: {local-name: ros_controllers, uri: 'https://github.com/ros-controls/ros_controllers.git', version: jade-devel}
-- git: {local-name: realtime_tools, uri: 'https://github.com/ros-controls/realtime_tools.git', version: jade-devel}
-- git: {local-name: control_toolbox, uri: 'https://github.com/ros-controls/control_toolbox.git', version: jade-devel}
-- git: {local-name: control_msgs, uri: 'https://github.com/ros-controls/control_msgs.git', version: jade-devel}
+# Note that for realtime_tools, control_toolbox and control_msgs there's no
```

Comment:
I'm not 100% this is the way to put comments here, but I'm almost 99% sure it's :)